src/main/java/com/mitchej123/hodgepodge/HodgepodgeEventHandler.java

```
@@ -19,4 +37,105 @@ public void onZombieAidSummon(ZombieEvent.SummonAidEvent event) {
+                                    1,
+                                    EnumMachineBeta.BOILER_FIREBOX_FLUID.ordinal()))) {
+                event.toolTip.add(
+                        String.format(
+                                "Produces %d Pollution/Second per firebox",
+                                Common.config.fireboxPollutionAmount));
+            }
+
+            if (GT_Utility.areStacksEqual(event.itemStack, EnumCart.BORE.getCartItem())) {
+                event.toolTip.add(String.format(producesPollutionFormat, Common.config.tunnelBorePollutionAmount));
+            }
+
+            if (GT_Utility.areStacksEqual(
+                    event.itemStack,
+                    new ItemStack(RailcraftBlocks.getBlockMachineAlpha(), 1, EnumMachineAlpha.COKE_OVEN.ordinal()))) {
+                event.toolTip.add(String.format(multiProducesPollutionFormat, Common.config.cokeOvenPollutionAmount));
+            }
+
+            if (GT_Utility.areStacksEqual(
+                    event.itemStack,
+                    new ItemStack(
+                            RailcraftBlocks.getBlockMachineAlpha(),
+                            1,
+                            EnumMachineAlpha.BLAST_FURNACE.ordinal()))) {
+                event.toolTip.add(
+                        String.format(multiProducesPollutionFormat, Common.config.advancedCokeOvenPollutionAmount));
+            }
+
+            if (GT_Utility.areStacksEqual(
+                    event.itemStack,
+                    new ItemStack(
+                            RailcraftBlocks.getBlockMachineBeta(),
+                            1,
+                            EnumMachineBeta.ENGINE_STEAM_HOBBY.ordinal()))) {
+                event.toolTip.add(String.format(producesPollutionFormat, Common.config.hobbyistEnginePollutionAmount));
+            }
+
+            if (Compat.isGalacticraftPresent() && Common.config.rocketsPollute) {
+                try {
```

Using `try` (with empty catch) to prevent crashing from `Objects.requireNonNull` isn't a good idea. I suggest to remove both and use regular null check.